docs/dev/debugging.rst

```diff
@@ -0,0 +1,40 @@
+Debugging
+=========
+
+
+.. _kerneloops:
+
+Kernel Oops
+-----------
+
+Sometimes a running Linux Kernel detects an error during runtime that can not
```

Comment:
"kernel" is usually written with a lowercase "k".

Reply:
Fixed. (I left the heading as it is.)